api/auth/success.html

```diff
@@ -1,123 +1,38 @@
-        color: #ff5500;
-        font-size: 32px;
-        font-weight: bold;
-        margin: 0;
-        margin-bottom: 10px;
-      }
-      .message {
-        color: #555;
-        font-size: 20px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        margin-top: 15px;
-      }
-      .message-link {
-        color: #ff5500;
-        text-decoration: none;
-        font-weight: bold;
-      }
-      .opensauced-logo {
-        display: flex;
-        align-items: center;
-        position: absolute;
-        top: 20px;
-        left: 20px;
-        color: #fff;
-        font-size: 28px;
-        font-weight: bold;
-      }
-      .logo-image {
-        width: 60px;
-        height: 60px;
-      }
-      .close-button {
-        position: absolute;
-        top: 10px;
-        right: 10px;
-        font-size: 24px;
-        color: #ff9900;
-        cursor: pointer;
-      }
-      .close-button:hover {
-        color: #ff5500;
-      }
-    </style>
-  </head>
-  <body>
-    <div class="container">
-      <div class="close-button" id="closeButton">&#10005;</div>
-      <div class="pizza-icon">🍕</div>
-      <h1 class="title">Authentication Successful</h1>
-      <p class="message" id="counter">
-        <span>You'll be redirected to </span>
-        <a href="https://insights.opensauced.pizza/" class="message-link"
-          >&nbspInsights&nbsp</a
-        >
-        <span> in 30 seconds</span>
-      </p>
-    </div>
-    <a href="https://opensauced.pizza" target="_blank" rel="noreferrer">
-      <div class="opensauced-logo">
-        <img
-          src="https://raw.githubusercontent.com/open-sauced/assets/main/logos/slice-White.png"
-          alt="OpenSauced Logo"
-          class="logo-image"
-        />
-        <div class="logo-text">OpenSauced</div>
-      </div>
-    </a>
-    <script>
-      const counterElement = document.getElementById("counter");
-      const closeButton = document.getElementById("closeButton");
 
-      let seconds = 30;
-      let interval = setInterval(() => {
-        if (seconds <= 1) {
-          clearInterval(interval);
-          window.open("https://insights.opensauced.pizza", "_self");
-        }
-        counterElement.innerHTML = `<span>You'll be redirected to </span>
-          <a href="https://insights.opensauced.pizza/" class="message-link">&nbspInsights&nbsp</a>
-          <span> in ${--seconds} second${seconds > 1 ? "s" : ""}</span>`;
-      }, 1000);
+<head>
+  <title>OpenSauced Pizza-CLI</title>
+  <style>
+    body {
+      margin: 1rem;
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+      background: black;
+      color: white;
+    }
+
+    .container {
+      display: grid;
+      place-content: center;
+      height: calc(100vh - 5.5rem);
+      overflow: hidden;
+      font-size: 1.5rem;
+    }
+  </style>
+  <meta http-equiv="refresh" content="10;url=https://opensauced.pizza/" />
```

Comment:
Should this redirect them to the `app.opensauced.pizza`? Landing page may not bring that much value for newly logged in users?

Reply:
oops, yep.